client/src/cb-admin/pages/VisitsData.js

```diff
@@ -109,7 +119,13 @@ export default class VisitsDataPage extends React.Component {
         this.props.updateAdminToken(resVisitors.headers.authorization);
 
         const visits = resVisitors.data.result;
-        const [visitsNumbers, genderNumbers, activitiesNumbers, ageGroups, activities] = resStats.data.result; //eslint-disable-line
+        const [
+          visitsNumbers,
+          genderNumbers,
+          activitiesNumbers,
+          ageGroups,
+          activities,
+        ] = resStats.data.result; //eslint-disable-line
```

Comment:
Is `\\eslint-disable-line` still needed here not that it spans multiple lines?

Reply:
Yeah you're right I've removed it